### PR TITLE
fix failing test

### PR DESCRIPTION
**Pull request by Aaron2000119**
fix for failing cloneTest

### CODE REVIEW

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java [hunk 1]
```diff
@@ -303,28 +323,36 @@ private Book cloneBook(final Book book) {
...
             case CROSSWORDPUZZLE:
-                return cloneCrosswordpuzzle(minigameTask, accessToken);
+                return cloneCrosswordpuzzle(minigameTask, accessToken, errorMessages);
             case BUGFINDER:
-                return cloneBugfinder(minigameTask, accessToken);
+                return cloneBugfinder(minigameTask, accessToken, errorMessages);
             default:
                 errorMessages.add(String.format("minigame %s doesn't exist", minigameTask.getGame()));
                 return new MinigameTask(Minigame.NONE, "", null, minigameTask.getIndex());
         }
```
Comment (maexled) [inline src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java]: towercrush was added in #56 . Make sure to resolve merge conflicts

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java [hunk 2]
```diff
@@ -348,17 +376,18 @@ private MinigameTask cloneBugfinder(final MinigameTask minigameTask, final Strin
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("bugfinder-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("bugfinder-backend not present");
-                    return new MinigameTask(Minigame.BUGFINDER, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```
Comment (maexled) [inline src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java:381]: In an exception, debug level? Also the backend name should be mentioned in the log.

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java [hunk 3]
```diff
@@ -382,17 +411,18 @@ private MinigameTask cloneCrosswordpuzzle(final MinigameTask minigameTask, final
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("crosswordpuzzle-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("crosswordpuzzle-backend not present");
-                    return new MinigameTask(Minigame.CROSSWORDPUZZLE, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```
Comment (maexled) [inline src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java:416]: In an exception, debug level? Also the backend name should be mentioned in the log.

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java [hunk 4]
```diff
@@ -411,17 +441,18 @@ private MinigameTask cloneFinitequiz(final MinigameTask minigameTask, final Stri
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("finitequiz-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("finitequiz-backend not present");
-                    return new MinigameTask(Minigame.FINITEQUIZ, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```
Comment (maexled) [inline src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java:446]: In an exception, debug level? Also the backend name should be mentioned in the log.

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java [hunk 5]
```diff
@@ -445,13 +476,10 @@ private MinigameTask cloneChickenshock(final MinigameTask minigameTask, final St
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("chickenshock-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("chickenshock-backend not present");
-                    return new MinigameTask(Minigame.CHICKENSHOCK, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```
Comment (maexled) [inline src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java:506]: In an exception, debug level? Also the backend name should be mentioned in the log.

**Review by maexled — COMMENTED** [review]
When I test it locally, these boths tests are still failing:
```
[ERROR]   CloneTest » ContainerLaunch Container startup failed
[ERROR]   CloneWithoutBackendsTest » ContainerLaunch Container startup failed
```

Any idea?

File: src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java (hunk 2 above)
Comment (Aaron2000119) [reply src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java:381]: this is changed in the refactored cloning so i won't change it here

**Review by maexled — APPROVED** [review]

**Pull request merged**